hello-world-notification-microservice/README.md

```diff
@@ -72,4 +76,69 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+
+## Prerequisite
+- cumulocity tenant with microservice hosting feature enabled
+  > Important: The Microservice hosting feature must be activated on your tenant, otherwise your request will return an error message like “security/Forbidden, access is denied”. This feature is not assigned to tenants by default, so trial accounts won’t have it. Contact [product support](https://cumulocity.com/guides/welcome/contacting-support/) so that we can assist you with the activation. Note that this is a paid feature. Microservice SDK for Java - Cumulocity IoT Guides
+- contents of this repository
+
+### Instructions
+1. Creating a test user
+    - From the Application Switcher select Administration
+    - In the Navigation pane expand Accounts and go to Users
+    - Click on 'Add user', fill in the form for the new user, untick "Send password reset link as email" and provide a password
```

Comment:
The user has two purposes - device authentication and to run the service under. Maybe say you can use an existing user instead but the role is needed.

Reply:
Unfortunately due to the authentication used on eu-latest you can't use the provided user - the same authentication details don't work while making REST requests, that's why I needed a separate user

Reply:
I'm totally confused about this now. I thought we said the service user works after all as we give it the ROLE in the microservice config? We still need a user with the role to "own" the microservice that the service user is substituting for.

Reply:
Happy to wait an retest once the nginx config is updated.

Reply:
no, this is different. when you create a trial account to try c8y on eu-latest you aren't presented with a regular login screen. instead it uses an authentication service ([link](https://env790224.eu-latest.cumulocity.com/apps/administration/index.html#/), the same credentials are useless if you want to do a REST call, so I thought the easiest would be to just a create another user under same tenant

Reply:
Oh, I see. Well no harm in creating a second user I guess.